redis/Cargo.toml

```diff
@@ -10,8 +10,36 @@ repository = "https://github.com/djc/bb8"
 [dependencies]
 async-trait = "0.1"
 bb8 = { version = "0.8", path = "../bb8" }
-redis = { version = "0.24", default-features = false, features = ["tokio-comp"] }
+redis = { version = "0.24" }
+# redis = { version = "0.24", default-features = false, features = [
+#   "tokio-comp",
+# ] }
 
 [dev-dependencies]
 futures-util = "0.3.15"
 tokio = { version = "1.0.0", features = ["macros", "rt-multi-thread"] }
+
+[features]
+default = ["redis/default"]
+acl = []
+aio = ["redis/aio"]
+geospatial = []
+json = ["redis/json"]
+cluster = ["redis/cluster"]
+script = ["redis/script"]
+tls-native-tls = ["redis/tls-native-tls"]
+tls-rustls = ["redis/tls-rustls"]
+tls-rustls-insecure = ["redis/tls-rustls-insecure"]
+tls-rustls-webpki-roots = ["redis/tls-rustls-webpki-roots"]
+async-std-comp = ["redis/async-std-comp"]
+async-std-native-tls-comp = ["redis/async-std-native-tls-comp"]
+async-std-rustls-comp = ["redis/async-std-rustls-comp"]
```

Comment:
Given that bb8 unconditionally relies on tokio, I think we should not expose the `async-std` features here.

Reply:
Change made